### ISSUE TITLE
SRFlow PyTorch Training: svd_cuda: the updating process of SBDSDC did not converge (error: 11)

### ISSUE THREAD

**Issue by flybiubiu** [andreas128/SRFlow#13]
Thx author!This is a great work!
But I train x4(python train.py -opt ./confs/SRFlow_DF2K_4X.yml) with problem.And with origin code.
Skipping ERROR caught in nll = model.optimize_parameters(current_step): 
svd_cuda: the updating process of SBDSDC did not converge (error: 11)

**Comment by andreas128** [comment]
Thanks, did the pretrained models work for you?

<br>

The error occurs when back-propagating thought the 1x1 convolution.

- Could you please post your training NLL from tensorboard?
- If you train on a different dataset, please try to reduce the learning rate.

**Comment by flybiubiu** [comment]
> Thanks, did the pretrained models work for you?
> 
> 
> The error occurs when back-propagating thought the 1x1 convolution.
> 
> * Could you please post your training NLL from tensorboard?
> * If you train on a different dataset, please try to reduce the learning rate.

Thx author!I resolve this problem. 
1.RTX 3090 with uncorrespond cuda and pytorch
2.conda with lots of uncorrespond library.I reinstall env with pip and everthing is ok.
Thx author
